components/NavBar/NavBar.module.css

```diff
@@ -7,10 +7,31 @@
   top: 0px;
   height: 60px;
 }
+.mobileTopNavContainer {
+  width: 100%;
+  background-color: #fff;
+  border-bottom: 1px solid #dbdbdb;
+  position: fixed;
+  left: 0;
+  z-index: 2;
+  top: 0px;
+  height: 44px;
```

Comment:
I don't think you need to set the height. If you're not happy with how it looks, adjust paddings and margins.

```suggestion
```